### DISCUSSION TITLE
PostgreSQL  PostGISQ FieldCloud and QField  how can they work toguether?

### DISCUSSION THREAD

**Discussion by jonywoker** [opengisch/QField#4718] [Q&A]
Let's consider a scenario where I have a server running PostgreSQL and PostGIS, and all users are utilizing QGIS to populate the spatial database. However, due to new requirements, there is a need to collect data in the field. Can QField, in conjunction with QFieldCloud, be used to gather information in the field and seamlessly integrate or transmit this data to the PostgreSQL server, ensuring that the new information is automatically updated for all users? If the answer is yes, where, and who do I contact to acquire this technical support, and what are the associated costs and time required for implementation?

**Comment by timbo-barnes** [comment]
Yes it is possible and there's help available to set this up here: https://docs.qfield.org/how-to/pg-service/

**Reply by jonywoker** [reply]
Thank you for your help
